perf/istio-install/create_sa.sh

```diff
@@ -46,17 +46,31 @@ PROJECT_ID=${PROJECT_ID:?"project id is required"}
 GCP_SA=${1:-istio-data}
 GCP_CTL_SA=${2:-istio-control}
 
-gc iam service-accounts create "${GCP_SA}" # --display-name 'Istio data plane account'
-gc iam service-accounts create "${GCP_CTL_SA}" #--display-name '"Istio control plane account"'
+if [[ -z "$(gcloud beta iam service-accounts --project="${PROJECT_ID}" list --filter=email="${GCP_SA}@${PROJECT_ID}.iam.gserviceaccount.com" --format='csv[no-heading](email)')" ]]; then
+  gc beta iam service-accounts create --quiet "${GCP_SA}" # --display-name 'Istio data plane account'
+fi
+if [[ -z "$(gcloud beta iam service-accounts --project="${PROJECT_ID}" list --filter=email="${GCP_CTL_SA}@${PROJECT_ID}.iam.gserviceaccount.com" --format='csv[no-heading](email)')" ]]; then
+  gc beta iam service-accounts create --quiet "${GCP_CTL_SA}" #--display-name '"Istio control plane account"'
+fi
+
+function gcloud_get_iam {
+  local sa="${1?"Must pass a SA as first argument to gcloud_get_iam function"}"
+  local role="${2?"Must pass a role as second argument to gcloud_get_iam function"}"
+  gcloud projects get-iam-policy "${PROJECT_ID}" --format=json | jq ".bindings[] | select((.members[] | contains(\"serviceAccount:${sa}@${PROJECT_ID}.iam.gserviceaccount.com\")) and .role == \"roles/${role}\")| .members[0]" -Mr
+
```

Comment:
When I run this locally, i got this command not found error, here is the log:

```
tools/perf/istio-install/create_sa.sh: line 59: jq: command not found
gcloud projects add-iam-policy-binding --quiet carolyntestprj-262400 --role roles/compute.admin --member serviceAccount:carolyn-test-v4-control@carolyntestprj-262400.iam.gserviceaccount.com
```

Reply:
https://stedolan.github.io/jq/ install this